content/blog/2018-03-23-hydroscoper.md

```diff
@@ -0,0 +1,157 @@
+## 358 min_envir_energy  24.45425 35.24414    298.6       kyy
+```
+The available time series from this station are:
+```r
+subset(timeseries, station_id %in% c(501032, 200280))
+
+##     time_id station_id       variable timestep units
+## 177     987     200280 wind_direction     <NA>     °
+## 279    2224     501032           flow     <NA>   l/s
+## 609     985     200280  precipitation    daily    mm
+## 668     986     200280           snow    daily    mm
+##                    start_date                  end_date subdomain
+## 177 1955-02-04T08:00:00+02:00 1997-03-31T09:00:00+03:00       kyy
+## 279 1970-09-01T00:00:00+02:00 2005-08-01T01:00:00+03:00       kyy
+## 609 1954-01-01T08:00:00+02:00 2015-09-30T09:00:00+03:00       kyy
+## 668 1954-01-01T08:00:00+02:00 1997-03-31T09:00:00+03:00       kyy
+```
+To download the daily precipitation time series run:
+```r
+prec <- get_data("kyy", 985)
+prec
+
+## # A tibble: 22,217 x 3
+##    date                value comment
+##    <dttm>              <dbl> <chr>  
+##  1 1954-01-01 08:00:00 15.3  <NA>   
+##  2 1954-01-02 08:00:00  5.60 <NA>   
+##  3 1954-01-03 08:00:00 10.2  <NA>   
+##  4 1954-01-04 08:00:00  0.   <NA>   
+##  5 1954-01-05 08:00:00 10.8  <NA>   
+## # ... with 22,212 more rows
+```
+Also, to download the flow time series run:
+```r
+flow <- get_data("kyy", 2224)
+flow
+
+## # A tibble: 324 x 3
+##    date                 value comment
+##    <dttm>               <dbl> <chr>  
+##  1 1970-09-01 00:00:00   3.86 <NA>   
+##  2 1970-10-01 00:00:00  10.1  <NA>   
+##  3 1970-11-01 00:00:00  25.8  <NA>   
+##  4 1970-12-01 00:00:00  32.1  <NA>   
+##  5 1971-01-01 00:00:00 100.   <NA>   
+## # ... with 319 more rows
+```
+Flow data seems to have a monthly time step. But, how someone can infer if these values are monthly averages or instantaneous? Let's create a plot!
+
+ * Firstly, subset the data to a narrower time window:
+
+```{r}
+min_date <- "1990-01-01 UTC"
+max_date <- "2000-01-01 UTC"
+flow <- subset(flow, date >= min_date & date <= max_date)
+prec <- subset(prec, date >= min_date & date <= max_date)
+```
+ * Finally, create the Precipitation Hyetograph and Stream-flow Hydrograph:
+ 
+```{r}
+library(gridExtra)
+library(ggplot2)
+
+g.top <- ggplot(prec, aes(x = date, y = value, ymin=0, ymax=value)) +
+  geom_linerange(col = "light blue") +
+  theme_minimal() +
+  theme(axis.text.x=element_blank())+
+  labs(y = "Prec. (mm)", x = "")
+
+g.bottom <- ggplot(flow, aes(x = date, y = value)) +
+  geom_line(color = "dark red") +
+  theme_minimal() +
+  labs(x = "Date", y = "Streamflow (l/s)")
+
+grid.arrange(g.top,g.bottom, heights = c(1/3, 2/3))
+```
+<center>
+<img src = "/img/blog-images/2018-03-23-hydroscope/hydrograph.png" style = "width:90%">
+</center>
+<p>
+
+The spikes in the plot indicate that the stream-flow values are monthly instantaneous. If there were monthly averages, one would expect a  smoother sequence of values.
```

Comment:
After this paragraph would be nice if you had a short summary or concluding statement, otherwise it seems to end abruptly (after a nice example). Maybe give verbal examples of one or two more helpful things one could plot or compare using hydroscoper